src/katcp_fpga.py

```diff
@@ -537,4 +537,19 @@ def unhandled_inform(self, msg):
         if self.unhandled_inform_handler is not None:
             self.unhandled_inform_handler(msg)
 
+    def check_phy_counter(self):
+        request_args = ((0, 0), (0, 1), (1, 0), (1, 1))
+        for arg in request_args:
+            try:
+                result0 = self.katcprequest('phywatch', request_args=arg)
+                time.sleep(1)
+                result1 = self.katcprequest('phywatch', request_args=arg)
+                assert result0[0].reply_ok()  # useful?
```

Comment:
Actually, the katcprequest() method already checks this by default and raises a KatcpRequestError exception. See the casperfpga.katcp_fpga.KatcpFpga.katcprequest method.